File: SELL3A Customer App/IOS/Sell3a/Model/Response/OrderResponse.swift
//
//  OrderResponse.swift
//  Sell3a
//
//  Created by Hager Samir on 26/06/2021.
//

import Foundation

// MARK: - ReviewResponse
struct OrderResponse: Codable {
    let itemIDS: [ItemID]
    let orderState: String
    let importCharge: Int
    let id, address, orderCode, orderDate: String
    let totalPrice: Int
    let userID: String
    let v: Int

    enum CodingKeys: String, CodingKey {
        case itemIDS = "itemIds"
        case orderState, importCharge
        case id = "_id"
        case address = "Address"
        case orderCode, orderDate, totalPrice
        case userID = "userId"
        case v = "__v"
    }
}

// MARK: - ItemID
struct ItemID: Codable {
    let color, companyName: String
    let count: Int
    let id, size: String
}

File: SELL3A Customer App/Android/Souq-development/app/src/main/java/ahmed/adel/sleeem/clowyy/souq/pojo/response/OrderResponse.kt
package ahmed.adel.sleeem.clowyy.souq.pojo.response

data class OrderResponse(
    val Address: String,
    val __v: Int,
    val _id: String,
    val importCharge: Int,
    val itemIds: List<ItemId>,
    val orderCode: String,
    val orderDate: String,
    val orderState: String,
    val totalPrice: Double,
    val userId: String
) {
    data class ItemId(
        val color: String,
        val count: Int,
        val companyName : String,
        val id: String,
        val size: String
    )
}